PHPUnit/Framework/TestResult.php

```diff
@@ -580,9 +580,13 @@ public function run(PHPUnit_Framework_Test $test)
         $errorHandlerSet = FALSE;
 
         if ($this->convertErrorsToExceptions) {
+            $errorLevels = E_ALL;
+            if (PHP_VERSION_ID < 70400) {
+                $errorLevels = E_ALL | E_STRICT;
+            }
```

Comment:
alternatives:

single line:
```php
$errorLevels = PHP_VERSION_ID < 70400 ? E_ALL | E_STRICT : E_ALL;`
```

multiline:
```php
$errorLevels = E_ALL;

if (PHP_VERSION_ID < 70400) {
    $errorLevels |= E_STRICT;
}

```

Reply:
I believe the current one is clear enough and easy to understand.